packages/cli/src/object/get.rs

```diff
@@ -23,6 +23,12 @@ impl Cli {
 			eprint!("∅");
 		}
 		eprintln!();
+		if let Some(depth) = metadata.depth {
+			eprint!("{depth}");
```

Comment:
You need to render a label here, see above.

Reply:
Done.